exercises/Tidyr-tree-biomass-R.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: exercise
+topic: Tidyr
+title: Tree Biomass
+language: R
+---
+
+Estimating the total amount of biomass (*the total mass of all individuals*) 
+in forests is important for understanding the global carbon budget and how the 
+earth will respond to increases in carbon dioxide emissions. Measuring the mass 
+of whole trees is a major effort and requires destructive harvest of the tree. 
+It's pretty much impossible to weigh an entire forest (*even if we were willing 
```

Comment:
Let's drop this sentence and the parenthetical. This stuff can be fun, but I think it makes the material more difficult for students whose first language isn't English.